app/webpacker/controllers/producers_select_controller.js

```diff
@@ -0,0 +1,37 @@
+import TomSelect from "./tom_select_controller";
+
+const minimumOptionsForSearchField = 11;
+
+export default class extends TomSelect {
+  static values = { options: Object, placeholder: String };
+
+  connect(options = {}) {
+    const template = this.template();
+    let additional_options = {};
+    const lines_of_option = template.content.children.length; 
+
+    if (lines_of_option < minimumOptionsForSearchField) {
+      additional_options.plugins = [];  
+    }
```

Comment:
~~I am not sure what this is doing, why do we need an empty `plugins` if we have less than 11 options ?~~
I re read the ticker,  it's to disable the search input. I think we should add a comment here

Reply:
I have added a comment here.